src/client/pages/Geo/GeoMap/GeoMapMenuData/MapVisualizerPanel/LayersSectionPanel/components/CustomAssetControl/CustomAssetControl.tsx

```diff
@@ -64,7 +74,7 @@ const CustomAssetControl: React.FC<Props> = ({
             onClick={() => onToggle(layerKey)}
             onKeyDown={() => onToggle(layerKey)}
           >
-            <div style={checked ? { backgroundColor } : {}} className={classNames('fra-checkbox', { checked })} />
+            {checkBoxContent}
```

Comment:
better to put this content inline. with ```{something && <></>}```
then classNames can be dynamic, so you need only one div checkbox, without duplicating it

Reply:
Done! I made only the classes and style dynamic, and the div is only defined once.